packages/wrangler/src/pages/upload.tsx

```diff
@@ -161,7 +169,7 @@ export const upload = async (
 
 	const fileMap = await walk(directory);
 
-	if (fileMap.size > 20000) {
+	if (fileMap.size > MAX_ASSET_COUNT) {
 		throw new FatalError(
 			`Error: Pages only supports up to 20,000 files in a deployment. Ensure you have specified your build output directory correctly.`,
```

Comment:
nit: would be nice to use `MAX_ASSET_COUNT` here as well instead of `20,000`. That way if we ever change the value we don't need to remember to update the message as well. Not sure if there'd be an easy way to format the value tho :/